### DISCUSSION TITLE
[@types/selectize]: Missing parameter "silent" in methods clearOptions() and refreshItems()

### DISCUSSION THREAD

**Discussion by AndreEckner** [DefinitelyTyped/DefinitelyTyped#69428] [Issues with a @types package]
There is a boolean|undefined parameter called silent in lots of the selecitze methods,
that handles whether the method should trigger a corresponding change-event.

This parameter exists as per now also in the two methods
clearOptions() and refreshItems().

Only for one of these cases is the parameter documented in the official API-Doc:
https://selectize.dev/docs/API/selectize#clearoptionssilent

For the missing documentation of refreshItems(silent) there currently exists an issue which nobody seems to care about:
https://github.com/selectize/selectize.js/issues/2009

But both cases can be seen in the current code at Github by searching for the method names:
https://github.com/selectize/selectize.js/blob/master/dist/js/selectize.js

So the Typescript-definitions should be changed accordingly to:
clearOptions(silent?: boolean): void;
refreshItems(silent?: boolean): void;

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in #69499
